### PR TITLE
Vagrant configuration file created

### PR DESCRIPTION
**Pull request by omersolmazer**
Vagrant configuration file is created and is ready for reviewing

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+  # Enable provisioning with a shell script. Additional provisioners such as
+  # Puppet, Chef, Ansible, Salt, and Docker are also available. Please see the
+  # documentation for more information about their specific syntax and use.
+
+  ######################################################################
+  # Add Python Flask environment
+  ######################################################################
+  # Forward Python Flask port
+  config.vm.network :forwarded_port, guest: 5000, host: 5000, auto_correct: true
```
Comment (p2hang) [inline Vagrantfile:57]: We may need to remove this for the IBM Bluemix deployment later.

**Review by p2hang — APPROVED** [review]
LGTM!

**Review by BobZcao — COMMENTED** [review]
This is a review

File: Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  ######################################################################
+  # Add Redis docker container
+  ######################################################################
+  config.vm.provision "shell", inline: <<-SHELL
+    # Prepare Redis data share
+    sudo mkdir -p /var/lib/redis/data
+    sudo chown vagrant:vagrant /var/lib/redis/data
+  SHELL
+
+  # Add Redis docker container
```
Comment (pinocchiochang) [inline Vagrantfile:78]: I don't think it's necessary to implement Redis for our project so far.

**Review by pinocchiochang — APPROVED** [review]
Approve to current Vagrant configuration file

**Review by BobZcao — APPROVED** [review]
Approve

**Pull request merged**